pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml

```diff
@@ -187,13 +185,7 @@ spec:
               AmazonCloudIntegration
             type: object
         type: object
-    served: true
+    served: false
```

Comment:
Is this right? Isn't this the served version of the API?

Reply:
I am unsure about this change, and not sure why the other CRDs all have this as `true`

Reply:
So I figured out what is going on. There is a v1 and a v1beta1 in this manifest. The v1 is served and the v1beta1 is not.